src/pynwb/ndx_icephys_meta/test/test_icephys.py

```diff
@@ -1243,7 +1244,7 @@ def test_add_icephys_meta_full_roundtrip(self):
         # check the id value
         self.assertEqual(res.index[0], 12)
         # Check that our sweep contains 1 IntracellularRecording
-        self.assertEqual(len(res.iloc[0]['recordings']), 1)
+        self.assertEqual(len(res), 1)
```

Comment:
@rly, while this is valid assert to make, it changes the meaning of what is being tested. The assert `` self.assertEqual(len(res), 1)`` checks that the table has a whole only a single row, whereas ``self.assertEqual(len(res.iloc[0]['recordings']), 1)`` tries to check whether the first cell in the column ``'recordings'`` contains a single element. The ``'recordings'`` column in this case is ragged array column and so the assert was trying to check whether the recording exists and that we have only 1. The same is true in line 1264, 1280, 1296, 1392, 1403, 1415, and 1426.